### PR TITLE
Main Readme

### PR DESCRIPTION
**Pull request by Arthi-chaud**
Why the ports were changed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,51 @@
...
+
+## What do I need to use Aeris?
+
+Make sure the following softwares are installed on your machine:
+
+- Docker, (and Docker-Compose)
+
+To setup Aeris, you need to provide the building system some information:
+
+- Client ID and Client Secrets for EACH services (fill the `.env.example`provided at the root of the repository)
```
Comment (Octopus773) [inline README.md:31]:
```suggestion
- Client ID and Client Secrets for EACH services (fill the `.env.example`provided at the root of the repository and rename it `.env`)
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,51 @@
...
+## How to install it ?
+
+To install Aeris, run the following commands at the root of the repository:
+
+- `docker-compose -f docker-compose.yml build` (grab a snack, it might take some time)
+- `docker-compose -f docker-compose.yml up`
+
+## How to use it ?
+
+The Aeris server is accessible on the host's 8080 port
```
Comment (Octopus773) [inline README.md:43]:
```suggestion
The Aeris server is accessible on the host's 81 port
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,51 @@
...
+
+You can access the Web client through port 8081
```
Comment (Octopus773) [inline README.md:45]:
```suggestion
You can access the Web client through `localhost`
```

File: README.md [hunk 4]
```diff
@@ -2,7 +2,51 @@
...
+
+An Android APK can be downloaded via localhost:8081/client.apk
```
Comment (Octopus773) [inline README.md:47]:
```suggestion
An Android APK can be downloaded via localhost/client.apk
```

File: README.md (hunk 3 above)
Comment (Arthi-chaud) [reply README.md:45]: This is not compliant to the subject: https://intra.epitech.eu/module/2021/B-YEP-500/NAN-5-1/acti-480798/project/file/B-YEP-500_area.pdf

**Review by Octopus773 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -2,7 +2,51 @@
...
+
+Aeris supports a variety of service, on which Pipelines are perfomed upon:
+
+- GitHub
+- Spotify
+- Youtube
+- Discord
+- Gmail
+- Twitter
+- Even more services in the future :smile:
```
Comment (AnonymusRaccoon) [inline README.md:21]: Please delete that.

Comment (AnonymusRaccoon) [reply README.md:21]: I don't think so.

File: README.md [hunk 6]
```diff
@@ -2,7 +2,51 @@
...
+
+To manage your Pipeline, you can use the Android App, as well as the Web client.
+
+Aeris supports a variety of service, on which Pipelines are perfomed upon:
+
+- GitHub
+- Spotify
+- Youtube
+- Discord
+- Gmail
```
Comment (AnonymusRaccoon) [inline README.md:19]: It's not gmail it's anilist now.

**Review by AnonymusRaccoon — APPROVED** [review]

**Pull request merged**